src/opnsense/mvc/app/controllers/OPNsense/Proxy/forms/main.xml

```diff
@@ -537,6 +537,24 @@
                 <allownew>true</allownew>
                 <advanced>true</advanced>
             </field>
+            <field>
+                <id>proxy.forward.icap.whiteList</id>
+                <label>Whitelist</label>
```

Comment:
I would rename that as "Whitelist" can be misleading. Basically ICAP is some kind of an remote procedure call protocol and not a ACL enforcement tool (even if it's most often used for that). I would prefer something like "Exclusion List" which fits the use case better (traffic will not be sent to the ICAP server).